### DISCUSSION TITLE
Border from terminal background shows through in VSCode interface

### DISCUSSION THREAD

**Discussion by donutbrew** [blinksh/blink#1729] [Support]
When on a non-black theme on the terminal side and you launch `code`, there is a border around the VSCode window that I think would be better color matched to the VSCode theme than the terminal theme.

See the image below—my terminal background is green and it “shows through” on the border:

![1E714E02-93FE-4DCE-8593-8C5944141B52](https://user-images.githubusercontent.com/6391757/204645499-c859f86e-f67f-4a89-9635-7868153f6781.jpeg)

**Comment by yury** [comment]
Set "Cover" layout mode. In 3 finger menu for current tab or in `config` -> `Appearance` for new tabs.
